README.md

```diff
@@ -163,23 +163,37 @@ EOF
 
 ## Suported Backends
 
-- [Vault](#vault)
-- [AWS SSM Parameter Store](#aws-ssm-parameter-store)
-- [AWS Secrets Manager](#aws-secrets-manager)
-- [GCP Secrets Manager](#gcp-secrets-manager)
-- [SOPS](#sops) powered by [sops](https://github.com/mozilla/sops))
-- [Terraform (tfstate)](#terraform-tfstate) powered by [tfstate-lookup](https://github.com/fujiwara/tfstate-lookup)
-- [Echo](#echo)
-- [File](#file)
+- [vals](#vals)
```

Comment:
I don't get the goal of this change. Perhaps you wanted to name this whole section like "ToC" and move it into the very beginning of this README? 😃

Reply:
My editor did that, reverted it.

Reply:
@stokkie90 Gotcha! Btw, which editor was it? I'm interested because it seems useful if used correctly.

Reply:
Used vscode. With a markdown extension.

Reply:
Will try. Thanks for the info!